KinomeScan-Appyter/Kinase Input Data Processing

```diff
@@ -0,0 +1,69 @@
+# Creates the dictionaries and converts them to GMT files
+
+import pandas as pd
+
+old_url = 'http://lincs.hms.harvard.edu/db/datasets/20000/results?small+molecules=HMS_ID&output_type=.csv'
+
+df = []
+
+kinome_df = pd.read_excel('http://lincs.hms.harvard.edu/wordpress/wp-content/uploads/2013/11/HMS-LINCS_KinomeScan_Datasets_2018-01-18.xlsx')
+kinome_df['sm_hms_id'] = kinome_df['sm_hms_id'].str.replace(r'\D', '') #remove HMSL before the ID
+
+# retrieve the CSV file for the small molecules, 2-183
+for x in range(0, 181):  
+    hms_id_string = str(kinome_df.iloc[x]['sm_hms_id'])
+    new_url = old_url.replace("HMS_ID", hms_id_string) # fills in HMS_ID with the correct small molecule 
```

Comment:
`format` is more robust and is made specifically for this:
`url = 'http://lincs.hms.harvard.edu/db/datasets/20000/results?small+molecules={0}&output_type=.csv'.format(hms_id_string)`